src/katsdpcontroller/tasks.py

```diff
@@ -430,7 +430,7 @@ def __init__(
             self._interface_sensors[request.network] = Sensor(
                 str,
                 self.name + f".interfaces.{request.network}.name",
```

Comment:
I like the mix of the new and the old :-P

Reply:
Dammit I didn't even notice the line above 😅